lightningd/plugin.c

```diff
@@ -154,9 +154,6 @@ struct plugin *plugin_register(struct plugins *plugins, const char* path TAKES,
 		if (streq(path, p_temp->cmd)) {
```

Comment:
unless i'm mistaken, this line will filter out non-matching plugins. only plugins with an identical path to the desired registree will be marked as important (if so indicated).

Reply:
Yeah you are right.